src/libs/actions/Policy.ts

```diff
@@ -2179,6 +2194,42 @@ function createWorkspaceFromIOUPayment(iouReport: Report | EmptyObject): string
     return policyID;
 }
 
+function setWorkspaceCategoryEnabled(policyID: string, categoriesToUpdate: Record<string, {name: string; enabled: boolean}>) {
+    const policyCategories = allCategoryPolicies?.[policyID] ?? {};
+
+    const onyxData: OnyxData = {
+        optimisticData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY_CATEGORIES}${policyID}`,
+                value: {
+                    ...Object.keys(categoriesToUpdate).reduce<Record<string, PolicyCategory>>((acc, key) => {
```

Comment:
```suggestion
                    ...Object.keys(categoriesToUpdate).reduce<Record<string, PolicyCategory>>((result, key) => {
```

Reply:
eslint doesn't allow for assignment, got ts error in this case, acc is an exception

Reply:
still don't know which rule is it but this error is weird 

<img width="764" alt="image" src="https://github.com/Expensify/App/assets/24796318/28043c4d-b6fe-4a69-9030-61caf2554019">